docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -57,79 +41,67 @@ namespace Prime.Services
+  <PackageReference Include="xunit" Version="2.2.0" />
+  <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+</ItemGroup>
 ```
 
-The test project requires other packages to create and run unit tests.
-`dotnet new` added xUnit, and the xUnit runner. You need to add the PrimeService
-package as another dependency to the project. You can do that using the `dotnet`
-CLI:
+The test project requires other packages to create and run unit tests. `dotnet new` in the previous step added xUnit and the xUnit runner. Now, add the `PrimeService` class library as another dependency to the project. Use the `dotnet` CLI:
 
 ```
 dotnet add reference ../PrimeService/PrimeService.csproj
 ```
 
-Or, you can directly edit the `PrimeService.Tests.csproj` file.
-Directly under the first
-`<ItemGroup>` node, add another `<ItemGroup>` node with a reference to 
-the library project:
+Another option is to edit the *PrimeService.Tests.csproj* file. Directly under the first `<ItemGroup>` node, add another `<ItemGroup>` node with a reference to the library project:
 
 ```xml
-  <ItemGroup>
-    <ProjectReference Include="..\PrimeService\PrimeService.csproj" />
-  </ItemGroup>
+<ItemGroup>
+  <ProjectReference Include="..\PrimeService\PrimeService.csproj" />
+</ItemGroup>
 ```
 
-You can see the entire file in the
-[samples repository](https://github.com/dotnet/docs/blob/master/samples/core/getting-started/unit-testing-using-dotnet-test/PrimeService.Tests/PrimeService.Tests.csproj) 
-on GitHub.
-
-After this initial structure is in place, you can write your first test.
-Once you verify that first unit test, everything is configured and should run smoothly
-as you add features and tests.
+You can see the entire file in the [samples repository](https://github.com/dotnet/docs/blob/master/samples/core/getting-started/unit-testing-using-dotnet-test/PrimeService.Tests/PrimeService.Tests.csproj) on GitHub.
 
 ## Creating the first test
 
-Before building the library or the tests, you need to run `dotnet restore`
-in both the `PrimeService` and `PrimeService.Tests` directories. This
-command restores all the necessary NuGet packages for each project.
+Before building the library or the tests, execute `dotnet restore` in the *PrimeService.Tests* directory. This command restores all the necessary NuGet packages for each project.
 
-The TDD approach calls for writing one failing test, then making it pass,
-then repeating the process. So, let's write that one failing test. Remove
-`UnitTest1.cs` from the `PrimeService.Tests` directory, and create a new
-C# file named `PrimeService_IsPrimeShould.cs` with the following content:
+The TDD approach calls for writing one failing test, making it pass, then repeating the process. Now, write one failing test: Remove *UnitTest1.cs* from the *PrimeService.Tests* directory and create a new C# file named *PrimeService_IsPrimeShould.cs* with the following content:
```

Comment:
> So, let's write that one failing test.

The conjunction and 1st person are problems. I can remove the whole sentence ... it isn't really needed.